sprints/dashboard/tasks.py

```diff
@@ -245,13 +299,10 @@ def complete_sprint_task(board_id: int) -> None:
                 future_next_sprint_number = create_next_sprint_task(board_id)
                 future_next_sprint = get_next_sprint(sprints, next_sprint)
 
-            cell_dict = {
-                'key': cell.key,
-                'name': cell.name,
-                'board_id': cell.board_id,
-            }
```

Comment:
I guess we don't need to change this anymore, right?

Reply:
@nizarmah, this was an unnecessary change, which was confusing, as this variable was not used anywhere earlier. Reverted in d916240.